Tests/DatadogTests/Datadog/TracerConfigurationTests.swift

```diff
@@ -69,6 +69,15 @@ class TracerConfigurationTests: XCTestCase {
         XCTAssertNil(tracingLogBuilder.carrierInfoProvider)
     }
 
+    func testDefaultTracerWithRUMEnabled() {
+        RUMFeature.instance = .mockNoOp()
+        defer { RUMFeature.instance = nil }
+
+        let tracer = Tracer.initialize(configuration: .init()).dd
+
+        XCTAssertNotNil(tracer.rumContextIntegration)
+    }
```

Comment:
I forgot to assert `nil` integration if `TracerConfiguration(..., bundleWithRUM: false, ...)` is passed. Will do before merge.

Reply:
Done https://github.com/DataDog/dd-sdk-ios/pull/245/commits/4c9ca9b538ee0ff4cac18023130a3eed85c6ac3b